src/utility/Format.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * 동적으로 문자열을 생성한다.
+ * @param {string} fString 대상 문자열 (ex. '제 이름은 {0}, 나이는 {1}세 입니다.')
+ * @param {Array} args 삽입할 문자열 배열 (ex. ['월하', 24])
+ * @return {string} 포맷팅된 문자열  (ex. '제 이름은 월하, 나이는 24세 입니다.')
+ */
+function FormatString(fString, args) {
+  return fString.replace(/{(\d+)}/g, (match, index) => {
+    return typeof args[index] !== 'undefined' ? args[index] : match;
+  });
+}
```

Comment:
```suggestion
function FormatString(template, args) {
  return template.replace(/{(\d+)}/g, (match, index) => args[index] ?? match);
}
```

?? 연산자를 사용하면 더 간결하게 표현이 가능한데 참고하시면 좋을거 같아여!

Reply:
유용한 문법을 배우게 되었네요! 감사합니다 :)

``` javascript
// return A ?? B

if (A !== 'undefined' && A !== null) {
  return A;
} else {
  return B;
}
```